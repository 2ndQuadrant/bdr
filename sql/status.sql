SELECT
  c::"char" AS status_char,
  bdr.node_status_from_char(c::"char") AS status_str,
  bdr.node_status_to_char(bdr.node_status_from_char(c::"char")) AS roundtrip_char
FROM (VALUES ('b'),('i'),('c'),('o'),('r'),('k')) x(c)
ORDER BY c;
