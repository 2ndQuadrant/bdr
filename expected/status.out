SELECT
  c::"char" AS status_char,
  bdr.node_status_from_char(c::"char") AS status_str,
  bdr.node_status_to_char(bdr.node_status_from_char(c::"char")) AS roundtrip_char
FROM (VALUES ('b'),('i'),('c'),('o'),('r'),('k')) x(c)
ORDER BY c;
 status_char |               status_str                | roundtrip_char 
-------------+-----------------------------------------+----------------
 b           | BDR_NODE_STATUS_BEGINNING_INIT          | b
 c           | BDR_NODE_STATUS_CATCHUP                 | c
 i           | BDR_NODE_STATUS_COPYING_INITIAL_DATA    | i
 k           | BDR_NODE_STATUS_KILLED                  | k
 o           | BDR_NODE_STATUS_CREATING_OUTBOUND_SLOTS | o
 r           | BDR_NODE_STATUS_READY                   | r
(6 rows)

